DuggaSys/microservices/courseedService/createNewCourse_ms.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+//---------------------------------------------------------------------------------------------------------------
+// editorService - Saves and Reads content for Code Editor
+//---------------------------------------------------------------------------------------------------------------
+
+// Missing Functionality
+//		New Code Example + New Dugga
+//		Graying link accordingly
+
+date_default_timezone_set("Europe/Stockholm");
+
+include('../shared_microservices/getUid_ms.php');
+
+getUid();
```

Comment:
what does this even do?

Reply:
we followed the guide

Reply:
well yes, but.
![image](https://github.com/HGustavs/LenaSYS/assets/102600391/f3849c5f-9bc8-4b2f-8389-31d2ab28322e)
you still need to know what your using and why it is used, that function did nothing at all useful in its original state. 
in its currents state (which i have talked over with its creator) it returns the userid of the current user. 

that said as it is, it does nothing 😥